datasafe-long-run-tests/datasafe-business-tests-random-actions/src/test/java/de/adorsys/datasafe/business/impl/e2e/randomactions/framework/BaseRandomActions.java

```diff
@@ -45,11 +46,19 @@
 
     private final Logger log = LoggerFactory.getLogger(getClass());
 
-    private static final int MEGABYTE_TO_BYTE = 1024 * 1024;
+    private static final int KILOBYTE_TO_BYTE = 1024;
     private static final long TIMEOUT = 30L;
 
-    private static final Set<Integer> THREAD_COUNT = ImmutableSet.of(2, 4);
-    private static final Set<Integer> FILE_SIZE_M_BYTES = ImmutableSet.of(1, 10);
+    private static String THREADS = readPropOrEnv("THREADS", "2, 4, 8");
+    private static String FILE_SIZES = readPropOrEnv("FILE_SIZES", "100, 1024, 10240"); // in KB
```

Comment:
Maybe fixture size can be read from env. var too, defaulting to some reasonable value

Reply:
ok